src/ThemeLoader.js

```diff
@@ -0,0 +1,98 @@
+import cssVarsPonyfill from 'css-vars-ponyfill';
+
+const buildURL = (cdn = '', themeId = '') => {
+    if (cdn === '' && themeId === '') return '';
+
+    if (cdn.charAt(cdn.length - 1) !== '/') {
+        return `${cdn}/${themeId}`;
+    }
+
+    return `${cdn}${themeId}`;
+};
+
+const isInternetExplorer11 = () => {
+    const ua = window.navigator.userAgent;
+    return ua.indexOf('Trident/') > 0;
+};
+
+const DEFAULT_LOADING_TIMEOUT = 2000;
+
+class ThemeLoader {
+    /**
+     * Loads the theme and the CSS vars ponyfill if necessary
+     *
+     * @param {Object} themeConfig - Object that contains the configuration to initialize the theme
+     * @param {string} themeConfig.cdn - Server URL that hosts the theme
+     * @param {string} themeConfig.themeId - Theme unique idenfitier
```

Comment:
maybe point it out here that the theme css is expected to live under cdn/themid.css